atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/H2ConnectionConfig.java

```diff
@@ -31,6 +34,13 @@
     public static final String TYPE = "h2";
 
     @Override
+    @JsonIgnore
+    @Value.Derived
+    public Optional<String> namespace() {
+        return Optional.of("H2 namespace");
```

Comment:
There's no restriction for any of the H2, InMemory or Jdbc namespaces, since neither of them are used.